Dockerfile

```diff
@@ -1,35 +1,26 @@
-FROM ubuntu:18.04
+FROM python:3.9.5-slim
```

Comment:
How does this impact the setup.cfg file? Is it supposed to replace `setup.cfg` or work with it?
- If the Docker works with `setup.cfg`, we should edit setup.cfg to specify python==3.9.5 so it works with the 3.9.5 docker image

Reply:
Hi Emily, I did some research and following are the reasons. 1. python 3.9.5-slim is more lightweight than standard ubuntu image, making it faster to pull, build and deploy. 2. python 3.9.5-slim comes pre-installed with Python 3.9.5 and pip, configured and ready to use.

Reply:
you may find that instead of using the python image, using ubuntu with anaconda ends up having faster performance. The reason for this is that anaconda uses numpy binaries that are better optimized for speed (from what I recall)